### PR TITLE
Lock sqlx-cli version in sqlx-db-check CI job

### LINKED ISSUE #2957: sqlx-db-check fails on master
I've created a few minor PRs that have nothing to do with the DB layer, nevertheless the DB check is failing. Some investigation is warranted.

### PR DESCRIPTION
**Pull request by klochowicz**
Fixes CI failure due to different latest version of the sqlx-cli tool and sqlx
crate in the workspace

### CODE REVIEW

**Review by luckysori — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -180,10 +180,10 @@ jobs:
       - name: Setup rust toolchain
         run: rustup show
       - uses: Swatinem/rust-cache@v2.0.0
-
       - name: Install sqlx-cli
+        # Lock sqlx-cli version to the same version as sqlx in the workspace
         run: |
-          cargo install sqlx-cli
+          cargo install sqlx-cli@0.6.1
```
Comment (bonomat) [inline .github/workflows/ci.yml:186]: Maybe lock it to the latest (0.6.2) and push the changes as well?

**Review by bonomat — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (klochowicz) [reply .github/workflows/ci.yml:186]: sure, but this makes sense only when sqlx gets upgraded. upgrading it now as a separate PR.

**Comment by klochowicz** [comment]
nice, 17 mins bors run!

**Pull request merged**